src/main/java/uk/org/openbanking/datamodel/payment/OBRisk1DeliveryAddress.java

```diff
@@ -58,8 +58,7 @@
   private String townName = null;
 
   @JsonProperty("CountrySubDivision")
-  @Valid
-  private List<String> countrySubDivision = null;
+  private String countrySubDivision = null;
```

Comment:
I'm still amazed OB have done this without creating a new object (i.e. `OBRisk2DeliveryAddress`).

There are countless objects that are all identification (e.g. `OBCashAccount1` to `OBCashAccount6`) when there was no need to create them and now we have this breaking change! 😕 😡